draft-wang-ppm-differential-privacy.md

```diff
@@ -974,15 +966,20 @@ one-hot vector, the L2-`SENSITIVITY` is `sqrt(2)`, because changing one
 coordinate of a Client measurement affects two coordinates. Algorithm 1 in
 {{BW18}} elaborates on the details of this tuning algorithm, and we will refer
 to the calculation in the open sourced code {{AGM}}.
+
 > JC: It's also worth exploring the utility of discrete Gaussian proven by
 > Definition 3 in {{CKS20}}, which defines the concentrated-DP achieved by
 > discrete Gaussian. Concentrated-DP is then converted to approximate-DP, which
 > is our target here. As a first draft, we won't overwhelm readers with other
 > types of DP.
 
-> KT: I think if we want to use BW18, we need some care as that applies to Gaussian and not to
-> discrete Gaussian. We can probably prove a version of that result for discrete Gaussian
-> or add continuous Gaussian noise and then round.
+> KT: {{CKS20}} mentions in Theorem 7 about the approximate-DP guarantee
```

Comment:
@kunal-talwar This is what I currently have in the [open PR](https://github.com/wangshan/draft-wang-ppm-differential-privacy/pull/25) for Aggregator randomization. Let me know if this captures your concern.